lib/matplotlib/tests/test_axes.py

```diff
@@ -4166,6 +4166,17 @@ def test_specgram_angle():
                                 scale="dB")
 
 
+def test_specgram_fs_none():
+    """ Test axes.specgram when Fs is None, should not throw error """
```

Comment:
The style checking is unhappy about the extra spaces around the docstring.

Reply:
Thanks! Fixed.